scormxblock/scormxblock.py

```diff
@@ -137,6 +137,12 @@ def author_view(self, context=None):
         frag = Fragment(html)
         return frag
 
+    @XBlock.handler
+    def get_scorm_template(self, request, suffix=''):
+        context_html = self.get_context_student()
```

Comment:
not sure we need to assign the variable when you use it only once, please do it like this
```suggestion
        template = self.render_template(
            'static/html/scorm_popup.html',
             self.get_context_student()
         )
```